index.js

```diff
@@ -74,6 +64,19 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      const hasConfig =
+        babelConfig.presets && babelConfig.presets.length > 0 ||
```

Comment:
>JS is not Ruby. 0 is falsy.

Correct me if I'm wrong, and it's not immediately clear, but this should work because @denysdovhan is checking `length > 0`.  

I suggest wrapping the comparison check in parens so it is more obvious that you're using a comparison operator.

```js
const hasConfig =
  babelConfig.presets && (babelConfig.presets.length > 0) ||
  babelConfig.plugins && (babelConfig.plugins.length > 0);
```